rust/PAC/Cargo.toml

```
@@ -0,0 +1,20 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "IBEX"
```

The convention is lowercase for package names

```suggestion
name = "ibex"
```